data/persistence/src/commonMain/sqldelightTimeline/social/androiddev/common/timeline/ConflictResolution.sq

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE failedWrite (
```

Comment:
What's this file/table for?

Reply:
When we start posting toots, we want to save failures (no network) so that we can retry them. This way we can eagerly  resolve show new local data and save it to DB without having to post to API. When network is back and we make a read we want to resolve any conflicts we might have. This table keeps track of keys/toots that we may have a conflict with. 

For now just a placeholder until I add the updating/posting logic. BTW this is based on the new androidx offline first guidance

Reply:
Got it, I dig it.